app/models/legal_aid_application.rb

```diff
@@ -148,8 +149,10 @@ def checking_answers?
       provider_checking_citizens_means_answers?
   end
 
+  # TODO: decide when is the submission_date set for passported and non-passported applications
+  # and save it to the DB
```

Comment:
In the CCMS payload, I've been assuming that the creation date of the CCMS::Submission record is the submission date.

Reply:
where do you get the submission date?